### PR TITLE
new-module survivor bedpetovcf

### PR DESCRIPTION
**Pull request by kubranarci**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [x] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/survivor/bedpetovcf/main.nf [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    tuple val(meta), path(bedpe)
+
+    output:
+    tuple val(meta), path("*.vcf"), emit: vcf
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
```
Comment (SPPearce) [inline modules/nf-core/survivor/bedpetovcf/main.nf:21]: Does the tool take any optional arguments at all? As this is currently missing the args that virtually all modules have

File: modules/nf-core/survivor/bedpetovcf/tests/main.nf.test [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                    ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    process.out.vcf,
```
Comment (SPPearce) [inline modules/nf-core/survivor/bedpetovcf/tests/main.nf.test:30]: Can just be process.out to snapshot all the channels in one line

File: modules/nf-core/survivor/bedpetovcf/tests/main.nf.test [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
```
Comment (SPPearce) [inline modules/nf-core/survivor/bedpetovcf/tests/main.nf.test:57]: Same here

**Review by SPPearce — COMMENTED** [review]
Looks generally fine, just a few comments

File: modules/nf-core/survivor/bedpetovcf/main.nf (hunk 1 above)
Comment (kubranarci) [reply modules/nf-core/survivor/bedpetovcf/main.nf:21]: This tool does not have any arguments.

**Comment by kubranarci** [comment]
thanks for the review @SPPearce 💯 Possible to 

> Looks generally fine, just a few comments

Thanks for the review!

File: modules/nf-core/survivor/bedpetovcf/tests/main.nf.test [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    process.out
+                ).match() }
```
Comment (SPPearce) [inline modules/nf-core/survivor/bedpetovcf/tests/main.nf.test:31]:
```suggestion
                { assert snapshot(process.out).match() }
```

File: modules/nf-core/survivor/bedpetovcf/tests/main.nf.test [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
```
Comment (SPPearce) [inline modules/nf-core/survivor/bedpetovcf/tests/main.nf.test:57]:
```suggestion
                { assert snapshot(process.out).match() }
```

**Review by SPPearce — APPROVED** [review]

**Comment by SPPearce** [comment]
Good to go, I've set to auto-merge.

**Pull request merged**